requirements.txt

```diff
@@ -29,4 +29,4 @@ nbmake
 
 # optional rt/test dependencies
 pennylane-lightning-kokkos
-amazon-braket-pennylane-plugin>1.27.1
+git+https://github.com/astralcai/amazon-braket-pennylane-plugin-python.git@qubit-device
```

Comment:
This change should be undone

Reply:
With `QubitDevice` removed from the `pennylane` top-level name space, the stable braket plugin will not work until they release a new version (the fix is in the latest braket plugin). Should I update this to install from master instead?

Reply:
This file is primarily meant to install build, develop, and test dependencies for developers. It is meant to be used once when setting up a dev environment. I think it is ok to leave as is, since we generally don't want to pull in a dev version of their environment, and if someone wants to run braket tests right now they can pull in the master version manually.

Reply:
Does CI install from the `requirements.txt` file? Which braket version is CI using?

Reply:
Looks like they just did.